src/java/htsjdk/tribble/TribbleIndexedFeatureReader.java

```diff
@@ -271,6 +273,27 @@ private void readHeader() throws IOException {
         return new WFIterator();
     }
 
+    //Visible for testing
+    static boolean isGZIPPath(final String path) {
```

Comment:
Should this method be moved to a utils class, instead of being buried inside here?